src/RedialContext.js

```diff
@@ -188,7 +192,10 @@ export default class RedialContext extends Component {
 
     Promise.all(promises)
       .then(this.props.onCompleted)
-      .catch((err) => this.props.onError(err, bail() || 'other'));
+      .catch((err) => {
+        this.props.onError(err, bail() || 'other');
+        this.abort(true);
+      });
```

Comment:
While I agree that we should abort (since we're no longer loading), I'm wondering whether we should also transition back to the previous location. If we abort, then we'll `setState`, which will cause a `render()`, which will render the newly matched location, but with data that failed to fetch. Perhaps that is not our concern, or maybe it is? Either way, I'd say it warrants further discussion